test/simple_table_pg/lib.rs

```diff
@@ -0,0 +1,6 @@
+pub mod models;
+pub mod schema;
+
+pub mod diesel {
+    pub use diesel::*;
+}
```

Comment:
I've noticed this in the other tests -- do you recall why this was necessary? Just curious :)

Reply:
see #94, #109 and then #116, which lead to the other PRs which changed to use `diesel::` basically everywhere (but not completely yet because of trait imports, see https://github.com/Wulf/dsync/issues/94#issuecomment-1850483291)

TL;DR: we are mostly there to completely remove this, but having this PR (#114) will make it a lot easier figuring out which traits need to be imported / the absolute path necessary